variables.sh

```diff
@@ -1,11 +1,9 @@
 #!/bin/bash
 
-set BASE_URL=http://localhost:8888/EcomSite/
-set BROWSER=chrome
-set RESULTS_DIR=$(pwd)/results
-set DB_PORT_OVERRIDE=8889
-set DB_HOST_OVERRIDE=localhost
-set DB_DATABASE_OVERRIDE=ecomsite
-set DB_TABLE_PREFIX_OVERRIDE=wp_
-set WOO_KEY=ck_b7504831538d902f98ebdb3c97e541412b1c3970
-set WOO_SECRET=cs_778cd37bb50c506ba42f964b9094a37cc21af3a3
\ No newline at end of file
+export BASE_URL=http://localhost:8888/EcomSite/
```

Comment:
You can keep 2 variable files. One for your local (Windows version) and one for Linux version.